src/components/time-picker/time-picker.tsx

```diff
@@ -222,15 +222,39 @@ export class TimePicker extends React.Component<Props, State> {
         }
     }
 
-    private changeValue(step: number): void {
-        const {currentSegment, ampm} = this.state;
+    private changeValue(step: number, isBigStep: boolean): void {
```

Comment:
lets call it `multiplier` and it would be 
```ts
private changeValue(step: number, multiplier: number = 1): void {}
```

Reply:
the idea is to pass boolean like `e.shiftKey`

Reply:
@tyv updated